src/kaleidoscope/plugin/LEDControl.cpp

```diff
@@ -19,7 +19,7 @@
 #include "kaleidoscope_internal/LEDModeManager.h"
 #include "kaleidoscope/keyswitch_state.h"
 
-using namespace kaleidoscope::internal;
+using namespace kaleidoscope::internal; // NOLINT(build/namespaces)
```

Comment:
Like in previous similar cases, we could technically make cpplint happy, and remove the `using namespace` construct, it wouldn't really buy us anything. As such, it's more practical to ignore the warning instead.